### DISCUSSION TITLE
More Vant Components 🚚

### DISCUSSION THREAD

**Discussion by lzm0x219** [3lang3/react-vant#185] [Ideas]
组件会持续更新下去，由于个人精力有限，所以速度可能不会太快，但是如果你有特别急的组件需求，欢迎在此留言，我会第一时间和你联系反馈👏。

推荐回复格式：
```bash
- 组件名称：
- vant链接：
- 额外特性：（如果需要）
```

**Comment by lzm0x219** [comment]
https://github.com/3lang3/react-vant/issues/5#issue-973196464

**Reply by HoPGoldy** [reply]
有考虑实现一个到达底部后上拉刷新的组件么？类似于 [PullRefresh](https://3lang3.github.io/react-vant/#/zh-CN/pull-refresh)。

我自己已经实现了相关的功能，可以的话我就提个 pr。

**Reply by 3lang3** [reply]
> 有考虑实现一个到达底部后上拉刷新的组件么？类似于 [PullRefresh](https://3lang3.github.io/react-vant/#/zh-CN/pull-refresh)。
> 
> 我自己已经实现了相关的功能，可以的话我就提个 pr。

非常欢迎啊😊
